queue/python/queue.py

```diff
@@ -0,0 +1,33 @@
+class Queue:
+    def __init__(self):
+        self.queue=[]
+
+    def is_empty(self):
+        return self.queue==[]
+
+    def enqueue(self, item):
```

Comment:
The code does not comply to PEP8.

*Origin: PEP8Bear, Section: `all.autopep8`.*

The issue can be fixed by applying the following patch:

```diff
--- a/tmp/tmpkyj0v92m/queue/python/queue.py
+++ b/tmp/tmpkyj0v92m/queue/python/queue.py
@@ -3,7 +3,7 @@
         self.queue=[]
 
     def is_empty(self):
-        return self.queue==[]
+        return self.queue == []
 
     def enqueue(self, item):
         self.queue.insert(0, item)
```